src/Basics.lidr

```diff
@@ -272,7 +279,7 @@ truth table -- for the `orb` function:
 -- TODO: Edit this
 
 We can also introduce some familiar syntax for the boolean operations we have
-just defined. The `syntax` command defines new notation for an existing
+just defined. The `syntax` command defines a new notation for an existing
```

Comment:
Actually, that's what the original has, see https://softwarefoundations.cis.upenn.edu/current/Basics.html: 
> We can also introduce some familiar syntax for the boolean operations we have just defined. The `Infix` command defines a new symbolic notation for an existing definition.